From: Patrick Bruenn <p.bruenn@beckhoff.com>
Date: Fri, 8 Aug 2025 11:16:38 +0200
Subject: BUILD: Accept krb5 1.22 for building the PAC plugin
Bug-Debian: https://bugs.debian.org/1122483
Forwarded: not-needed
Origin: upstream, https://github.com/SSSD/sssd/commit/38a6a4a79226a784eb2233203f74cf2040e67856

At least on FreeBSD 14.2 this fixes the build with krb5 1.22.

Signed-off-by: Patrick Bruenn <p.bruenn@beckhoff.com>

Reviewed-by: Sumit Bose <sbose@redhat.com>
---
 src/external/pac_responder.m4 | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/external/pac_responder.m4 b/src/external/pac_responder.m4
index 90727185b..3501b6b71 100644
--- a/src/external/pac_responder.m4
+++ b/src/external/pac_responder.m4
@@ -23,7 +23,8 @@ then
         Kerberos\ 5\ release\ 1.18* | \
         Kerberos\ 5\ release\ 1.19* | \
         Kerberos\ 5\ release\ 1.20* | \
-        Kerberos\ 5\ release\ 1.21*)
+        Kerberos\ 5\ release\ 1.21* | \
+        Kerberos\ 5\ release\ 1.22*)
             krb5_version_ok=yes
             AC_MSG_RESULT([yes])
             ;;
-- 
2.47.3


